sdk/tests/core/test_az_mqtt5_connection.c

```diff
@@ -109,44 +106,27 @@ static az_result test_subclient_policy_1_root(az_event_policy* me, az_event even
       ref_conn_close_req++;
       break;
     case AZ_HFSM_EVENT_ERROR:
-      ref_conn_error++;
-      break;
-    case AZ_MQTT5_EVENT_CONNECT_RSP:
-    {
-      az_mqtt5_connack_data* data = (az_mqtt5_connack_data*)event.data;
```

Comment:
Do these connack reasons still need to be checked?

Reply:
Not anymore, because of that the subclient also has `assert(false)` as a default

Reply:
Not anymore, open and failed connections are sent through the new indicator events